### PR TITLE
Change dsmr.markdown for configuration flow and add new logo

### PR DESCRIPTION
**Pull request by RobBie1221**
## Proposed change

Add to the DSMR integration two new parameters, force_update and prefix

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37464
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1739
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/dsmr.markdown [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
 title: DSMR Slimme Meter
 description: Instructions on how to integrate DSMR Smartmeter within Home Assistant.
-logo: netbeheernederland.jpg
+logo: netbeheernederland.png
```
Comment (klaasnicolaas) [inline source/_integrations/dsmr.markdown]: Please add a new brand logo to this repository -> https://github.com/home-assistant/brands and remove the line here

File: source/_integrations/dsmr.markdown [hunk 2]
```diff
@@ -46,10 +47,20 @@ DIY solutions (ESP8266 based):
 
 - [esp8266_p1meter (fliphess)](https://github.com/fliphess/esp8266_p1meter)
 
+## Configuration
+
+There are two ways to integrate your DSMR into Home Assistant
+
+### Via the frontend
+
+Menu: *Configuration* -> *Integrations*. Search for "DSMR Slimme Meter" and follow the configuration flow.
```
Comment (klaasnicolaas) [inline source/_integrations/dsmr.markdown]:
```suggestion
Menu: **Configuration** -> **Integrations**. Search for "DSMR Slimme Meter" and follow the configuration flow.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/dsmr.markdown (hunk 1 above)
Comment (RobBie1221) [reply source/_integrations/dsmr.markdown]: Just double checking, I already added brand logo's: https://github.com/home-assistant/brands/pull/1739
So basically if I remove the logo item here it'll automatically pick up from the brands repository?

I also removed the jpg and added a new logo as png in the supported_brands folder, Should I just remove the jpg then from this folder? I guess it's nowhere used anymore after removing the line.

Comment (klaasnicolaas) [reply source/_integrations/dsmr.markdown]: Yes, in that case you only have to delete this line and the .png file, everything is retrieved from the brand repository 😉

Comment (RobBie1221) [reply source/_integrations/dsmr.markdown]: Ok, clear, removed it

File: source/_integrations/dsmr.markdown [hunk 3]
```diff
@@ -46,10 +46,20 @@ DIY solutions (ESP8266 based):
...
+Menu: **Configuration** -> **Integrations**. Search for "DSMR Slimme Meter" and follow the configuration flow.
+
+### Via the configuration file
+
 ```yaml
 # Example configuration.yaml entry
-sensor:
-  - platform: dsmr
+dsmr:
+  port: /dev/ttyUSB0
```
Comment (frenck) [inline source/_integrations/dsmr.markdown]: We should not introduce new YAML options for integrations that communicate with a device or server.

For more information see ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/dsmr.markdown (hunk 3 above)
Comment (RobBie1221) [reply source/_integrations/dsmr.markdown]: Yeah, I get your point. The idea behind it was to go easy on people who currently use yaml and have an easy way to rewrite the configuration to a domain (not introduce new options).

I have no problems deprecating yaml alltogether and only allow UI configuration.

Comment (RobBie1221) [reply source/_integrations/dsmr.markdown]: I have to say at the moment I made this I took examples from the docs and core and saw this as a nice solution in line with existing integrations, but I guess there is a lot of legacy around especially as ADR-0010 is not that old.

I'll remove yaml support from the docs and from the core PR that is linked to this PR

**Comment by RobBie1221** [comment]
Removed YAML references from the docs

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**